staging/src/k8s.io/client-go/tools/remotecommand/remotecommand.go

```diff
@@ -48,6 +49,9 @@ type Executor interface {
 	// is set, the stderr stream is not used (raw TTY manages stdout and stderr over the
 	// stdout stream).
```

Comment:
should this be marked deprecated in favor of StreamContext?

Reply:
Yeah. I think so too. Will be marked.

Reply:
@liggitt I've mark it as deprecated in a new commit. Please review again when you have time. Thanks.